app/controllers/pageflow/editor/files_controller.rb

```diff
@@ -74,6 +93,7 @@ def destroy
         authorize!(:edit, entry.to_model)
         verify_edit_lock!(entry)
         entry.remove_file(file)
+        file.destroy if file.uploadable?
```

Comment:
There already is some [logic around removing files in `DraftEntry`](https://github.com/codevise/pageflow/blob/1f7c77d3da2978a9b67e60182f401cebcc83e199/app/models/pageflow/draft_entry.rb#L60). Maybe this can go (and be tested) there as well?

Reply:
True, the logic is there and tested so no need to do it here.
Problem is: [the code] (https://github.com/codevise/pageflow/blob/1f7c77d3da2978a9b67e60182f401cebcc83e199/app/models/pageflow/draft_entry.rb#L60) didn't work as expected: The usages are destroyed via the draft, but the file doesn't know about it at this point, so it never got deleted. A simple `reload` on the usages does the trick, though